### ISSUE TITLE
Play a sound when the sound timer is greater than 0

### ISSUE THREAD

**Issue by Taiters** [Taiters/chip8-web#7]
Currently the sound timer is implemented, and counts down to 0 when an instruction modifies it, however no sound currently plays. The intended functionality of the sound timer is to play a sound whilst the timer is above 0. This sound can be any tone using an [OscillatorNode](https://developer.mozilla.org/en-US/docs/Web/API/OscillatorNode) for example.

A good place to start to look into this would be in [cpu.js](src/cpu/cpu.js).

**Comment by eugeniumegherea** [comment]
What do you mean by 

> however no sound currently plays

Do you mean a proper bit music or just beeps when it decrements?
I started looking into it

**Comment by Taiters** [comment]
Hey @eugeniumegherea, thanks for taking a look!

Yeah it's just a beep. There's an explanation in the **Timers** section on this page: http://mattmik.com/files/chip8/mastering/chip8.html

So a sine wave etc would work.

**Comment by eugeniumegherea** [comment]
Could you take a look before I make a PR? I've implemented a beep class which plays every 16ms. I was not sure if it should play continuously until it's not 0 or play at each frame. So I made that it plays for each frame

**Comment by Taiters** [comment]
Awesome! I'll try it locally now

**Comment by Taiters** [comment]
Yeah that works really well, actually sounds pretty cool/retro when it plays a short burst per frame
